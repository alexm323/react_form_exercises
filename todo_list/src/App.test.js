import { render } from '@testing-library/react';
import App from './App';

test('renders without crashing', () => {
  render(<App />)
})

test('snapshot test', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
})