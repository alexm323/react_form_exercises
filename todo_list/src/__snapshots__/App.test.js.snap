// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <label
          forhtml="todo"
        >
          todo
        </label>
        <input
          aria-label="todo"
          id="todo"
          name="todo"
          type="text"
          value=""
        />
        <button>
          Add Todo
        </button>
      </form>
      <h2>
        Todo list:
      </h2>
      <ul />
    </div>
  </div>
</DocumentFragment>
`;
